config/desktop.json

```diff
@@ -82,6 +82,7 @@
 		"settings/theme-setup": false,
 		"signup/domain-first-flow": true,
 		"signup/social": false,
+		"standalone-site-preview": true,
```

Comment:
just curious -- did we test in desktop?

We can test after this is merged by bumping the submodule if you'd like since we have the luxury of waiting for a release.

Reply:
I tried but failed on build step of desktop app… :( 

However, I don't see any problem that might happen on the desktop. We even have more compatibility with sites, as we don't need to honor x-frame-options and https rules there.